content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+Hello Jellyfin users out there! Welcome back to our blog!
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
+**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
+the interface you access using your browser.
+
+Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!
+
+We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
+We hope you will forgive us (*send everyone who contributed a cake if you do, please!*) after trying it!
+
+# Key Features
+
+## Let your content shine like never before
+
+If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted into this, so I it seems that, at least, they want to:
```

Comment:
Maybe something like this:

> If you're like us, you probably want your content **to be** showcased as **beautifully** as possible, right? Our volunteers working on the web interface seem to agree, and have been very committed to this goal:

Reply:
I think that contributors is better, basically because it's the widely known term for people who work in open source software. At least based on what I've seen in other places.